### PR TITLE
feat(*): add Taiwan and Singapore Cloud Function Locations

### PR DESCRIPTION
**Pull request by thatfiredev**
This PR should:
- update firebase-admin to `9.11.0`
- update firebase-functions to `3.15.3`
- add Taiwan (`asia-east1`) and Singapore (`asia-southeast1`) to the list of available Cloud Function Locations for every extension.

### CODE REVIEW

File: rtdb-limit-child-nodes/functions/package.json [hunk 1]
```diff
@@ -11,8 +11,8 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "firebase-admin": "^8.0.0",
-    "firebase-functions": "^3.13.2"
+    "firebase-admin": "^9.11.0",
```
Comment (dackers86) [inline rtdb-limit-child-nodes/functions/package.json:14]: Updating these references causes the following build issues....

```js
src/index.ts(32,33): error TS2339: Property 'path' does not exist on type 'Reference'.
src/index.ts(43,39): error TS2339: Property 'path' does not exist on type 'Reference'.
src/index.ts(45,38): error TS2339: Property 'path' does not exist on type 'Reference'.
src/index.ts(47,36): error TS2339: Property 'path' does not exist on type 'Reference'.
```

It seems path was removed at some point before the `v9` release

**Review by dackers86 — CHANGES_REQUESTED** [review]

**Comment by dackers86** [comment]
hi @rosariopfernandes.

Following the above feedback, would it be possible  to remove the dependency updates and to just include the additional location code changes for this PR.

In addition, could you create a separate PR for the dependency updates, this allows a seperation of concerns for the PRs and would allow this particular PR to be quickly approved.

Thanks!

**Comment by rosariopfernandes** [comment]
Hey @dackers86
Sorry for the delayed response here. Had a busy week and missed the notifications for this PR...
I see you've already sent separate PRs to fix the build here. Thanks for that!
Is there any action still required from me?

**Comment by dackers86** [comment]
HI @rosariopfernandes 

Apologies, I missed your message! No more action needed, in terms of the updated dependencies for the extension modules we have decided to update these as separate PRs as noted above. 

Updating the dependencies can be moved to the individual PRs so the location update is now ready for the next release.

**Comment by dackers86** [comment]
Adding to blocking column. Happy to keep this open as a reference to the linked PRs.

**Comment by dackers86** [comment]
Closing as configurable location params have now been removed from extensions in this repository.

**Pull request closed without merging**